invenio_app_ils/records/api.py

```diff
@@ -408,9 +416,23 @@ def get_patron(patron_pid):
         if not patron_pid:
             raise PatronNotFoundError(patron_pid)
 
+        if str(patron_pid) == "-1":
```

Comment:
After the change below:
```suggestion
        if str(patron_pid) == str(SystemAgent.id):
```